static/js/query-configs/websites.ts

```diff
@@ -235,57 +236,82 @@ export const createWebsiteCollaboratorMutation = (
   }
 }
 
-export type WebsiteContentListingResponse = CollectionResponse<
+export type WebsiteContentListingResponse = PaginatedResponse<
   WebsiteContentListItem
 >
 type WebsiteContentListing = Record<string, string[]> // website name to list of uuids
 export const contentListingKey = (
-  name: string,
-  type: string,
-  offset: number
-): string => JSON.stringify([name, type, offset])
+  listingParams: ContentListingParams
+): string => {
+  const key = [
+    listingParams.name,
+    listingParams.type,
+    listingParams.subtype,
+    listingParams.offset
+  ]
+  if (listingParams.limit) {
+    key.push(listingParams.limit)
```

Comment:
Can we add or modify a test so that it covers this line?

Reply:
This and the other lines you commented on in this file are tested by `static/js/components/SiteContentListing.test.tsx`

Reply:
I'm not sure, if I add a `throw new Error` at this line it doesn't change any test output. Same for the other lines

Reply:
That's odd... we're mocking the API request to return certain data given the URL with those extra parameters. I'll look into it